dfstools/src/relationship_tools.py

```diff
@@ -1,6 +1,37 @@
 import pandas as pd
 
 
+def pecan_cookies_load_data():
+    pd.set_option("display.max_rows", None, "display.max_columns", None)
+    pd.set_option('display.width', 15000)
+
+    airports = pd.read_csv("./data/airports/airports.csv")
+    airlines = pd.read_csv("./data/airlines/airlines.csv")
+    flights = pd.read_csv("./data/flights/flights.csv")
+
+    dataframe_list = {
+        'airports': airports,
+        'airlines': airlines,
+        'flights': flights
+    }
+
+    return dataframe_list
+
+
+def find_similar_data(current_table, current_col, dataframe_dict):
+    relationship_list = []
+
+    target_set = set(dataframe_dict[current_table][current_col])
+    for table in dataframe_dict:
+        if table != current_table:
+            for col in dataframe_dict[table]:
+                compare_set = set(dataframe_dict[table][col])
+                if target_set.issubset(compare_set) or target_set.issuperset(compare_set):
```

Comment:
Have you had a chance yet to do any research on how to improve performance here?

Reply:
Hey! thanks for the quick review! and yes! I am still working on that in a separate file on my local machine. I'll make a PR for that once I finish the optimization.